README.md

```diff
@@ -34,7 +34,7 @@ SmartRedis provides clients in the following languages:
 
 | Language   | Version/Standard                               |
 |------------|:----------------------------------------------:|
-| Python     |   3.8, 3.9, 3.10, 3.11                         |
+| Python     |   3.9, 3.10, 3.11                         |
 | C++        |   C++17                                        |
```

Comment:
Very pedantic style thing, but my OCD will not forgive me if I do not call it out: Can we realign these pipes, lol?
```suggestion
| Python     |   3.9, 3.10, 3.11                              |
| C++        |   C++17                                        |
```

Reply:
definitely!